### PR TITLE
Adding Drizly to workflow menu

### PR DESCRIPTION
**Pull request by emily-hawkins**
Added docs for drizly's Github actions workflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # sqlfluff-github-actions
 The official resource for SQLFluff related [GitHub Actions and Workflows](https://docs.github.com/en/actions).
 
-## Menu of Github Workflows
-Workflows are listed below by contributing team with a brief description. To learn more and how to implement each click links below.
+## Menu of GitHub Workflows
```
Comment (barrywhart) [inline README.md:4]: Attention to detail = 💯 🎉

File: menu_of_workflows/drizly/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Developed by Emily Hawkins @ Drizly
+
+To add this to your repo, copy the contents of `lint_sqlfluff.yml` in this folder into a file named `.github/workflows/lint_sqlfluff.yml`.
+
+## Assumptions
+1. Only SQL files in your `dbt/models` directory are linted
+2. You are on the latest version of SQLFluff (at least 0.6.1) with the format `github-annotations` available.
+
+## Usage notes
+This action will take any changed files (modifed or added) in a PR and lint them using your SQLFluff configuration.
```
Comment (barrywhart) [inline menu_of_workflows/drizly/README.md]:
```suggestion
This action will take any changed files (modified or added) in a PR and lint them using your SQLFluff configuration.
```

**Review by barrywhart — APPROVED** [review]
Looks great!

I noticed one little typo we may want to fix before merging.

File: menu_of_workflows/drizly/README.md (hunk 2 above)
Comment (emily-hawkins) [reply menu_of_workflows/drizly/README.md]: fixed! thank you

**Comment by emily-hawkins** [comment]
i dont have write access to this repo, so whoever can, feel free to merge when you feel good about this!

**Pull request merged**